systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaConnectResource.java

```diff
@@ -87,8 +91,8 @@ private static KafkaConnectBuilder defaultKafkaConnect(KafkaConnect kafkaConnect
             .endSpec();
     }
 
-    private static DoneableKafkaConnect deployKafkaConnect(KafkaConnect kafkaConnect) {
-        if (Environment.DEFAULT_TO_DENY_NETWORK_POLICIES) {
+    private static DoneableKafkaConnect deployKafkaConnect(KafkaConnect kafkaConnect, boolean allowNP) {
+        if (Environment.DEFAULT_TO_DENY_NETWORK_POLICIES && allowNP) {
             KubernetesResource.allowNetworkPolicySettingsForResource(kafkaConnect, KafkaConnectResources.deploymentName(kafkaConnect.getMetadata().getName()));
         }
         return new DoneableKafkaConnect(kafkaConnect, kC -> {
```

Comment:
```suggestion
    private static DoneableKafkaConnect deployKafkaConnectWithNetworkPolicy(KafkaConnect kafkaConnect) {
        if (Environment.DEFAULT_TO_DENY_NETWORK_POLICIES && allowNP) {
            KubernetesResource.allowNetworkPolicySettingsForResource(kafkaConnect, KafkaConnectResources.deploymentName(kafkaConnect.getMetadata().getName()));
        }
        return deployKafkaConnect(kafkaConnect)
      }

    private static DoneableKafkaConnect deployKafkaConnect(KafkaConnect kafkaConnect) {
        return new DoneableKafkaConnect(kafkaConnect, kC -> {
```
This is cleaner in my opinion, boolean arguments are the devil

Reply:
A different method signature is preferable to passing true/false